mcodingbot/plugins/peps.py

```diff
@@ -89,19 +127,33 @@ def within_age_cutoff(message_created_at: datetime) -> bool:
     return datetime.now(timezone.utc) - message_created_at <= MAX_AGE_FOR_SEND
 
 
-def get_peps_embed(content: str) -> hikari.Embed | None:
-    pep_refs = [
-        int(ref.group("pep")) for ref in re.finditer(PEP_REGEX, content)
-    ]
-    peps = map(pep_manager.get, sorted(set(pep_refs))[:5])
-    pep_links_message = "\n".join(str(pep) for pep in peps if pep)
+def get_pep_refs(content: hikari.UndefinedNoneOr[str]) -> list[PEPInfo]:
+    """
+    Return a sorted list of all the peps mentioned in a string.
+    """
+    if not content:
+        return []
+
+    peps = sorted(
+        [int(ref.group("pep")) for ref in re.finditer(PEP_REGEX, content)]
+    )
```

Comment:
Useless list comprehension + I don't see the point of `re.finditer` instead of using the pattern directly
```suggestion
    peps = sorted(int(ref.group("pep")) for ref in PEP_REGEX.finditer(content))
```

Reply:
I just moved this didn't write it lol. Commiting your suggestion.